tests/test_api.rs

```diff
@@ -8690,6 +8690,7 @@ fn icu_date() {
   }
 }
 
+#[ignore]
```

Comment:
Why ignoring this test?

Reply:
Oh, this test is actually causing UB in the C code. It reads from undefined bytes. I don't have a great way to detect ASAN right now.